apps/internal/services/fixture_factory.py

```
@@ -0,0 +1,276 @@
+from apps.tasks.models import TaskLog, Error
+from fyle_accounting_library.common_resources.models import DimensionDetail
+from fyle_accounting_mappings.e2e_fixtures import BaseFixtureFactory
+
+class FixtureFactory(BaseFixtureFactory):
+    """Factory for creating Intacct test fixture data"""
+
+    def create_dependent_field_settings(self, workspace, count=2):
+        """Create sample dependent field settings"""
+        settings = []
+
+        for i in range(count):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)